ios/FluentUI/ResizingHandleView/ResizingHandleTokens.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+/// Design token set for the `ResizingHandleView` control.
+open class ResizingHandleTokens: ControlTokens {
+    /// Defines the color of the mark of the `ResizingHandle`.
+    open var markColor: DynamicColor {
+        // TODO: Update to aliasTokens.strokeColors[.neutral1] with Fluent 2 colors
```

Comment:
Do we have this work tracked in ADO? Probably easier to better to manage this work in ADO and remove the TODO comments in the code (as the Fluent2 work will likely happen after this is in the main branch).